src/components/00-materials/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.0.0
+
+- BREAKING CHANGE: The icons `arrow-left`, `arrow-right`, `collapse`, `document`, `download`, `email`, `expand`, `mobile`, `phone`, `plus`, `search`, `upload`, `cloud-upload` have been updated and appear a bit smaller. (#1725)
+
```

Comment:
please put `cloud-upload` in the right alphabetical order

Reply:
agree as a minimal improvement

Reply:
🤣 my man!